BUILD.md

```diff
@@ -187,7 +187,7 @@ See more information on the TensorRT Execution Provider [here](./docs/execution_
    * The path to the CUDA `bin` directory must be added to the PATH environment variable so that `nvcc` is found.
    * The path to the cuDNN installation (path to folder that contains libcudnn.so) must be provided via the cuDNN_PATH environment variable, or `--cudnn_home parameter`.
  * Install [TensorRT](https://developer.nvidia.com/nvidia-tensorrt-download)
-   * The TensorRT execution provider for ONNX Runtime is built and tested with TensorRT 6.0.1.5.
+   * The TensorRT execution provider for ONNX Runtime is built and tested with TensorRT 6.0.1.5. Note: Some TensorRT 6.0.1.5 new features such as dynamic shape is not supported in this release and will be supported soon after being fully tested.
```

Comment:
change ...built and tested with TRT 6.0.1.5 __but validated with the feature set equivalent to TRT 5. Some new features in TRT 6 like dynamic shape is not available as this time.__
Need to also update TensorRT-ExecutionProvider.md about this limitation for supporting dynamic shape.